### PR TITLE
add language in linlang files

### PR DESCRIPTION
**Pull request by SoranTabesh**
### Feature Preview

---

#### PR Checklist

- [x] I added a line related to the Kurdish (Sorani) language to the inlang files.

### CODE REVIEW

**Comment by richardshiue** [comment]
Hi @SoranTabesh thanks for contributing once again! Please resolve the conflicts in the PR so that we can get this PR merged

File: frontend/appflowy_flutter/packages/flowy_infra/lib/language.dart [hunk 1]
```diff
@@ -22,6 +22,11 @@ String languageFromLocale(Locale locale) {
       return "العربية";
     case "ca":
       return "Català";
+    case "ckb":
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/packages/flowy_infra/lib/language.dart:25]: There's already a `ckb-KU` code in the main branch. Please check which one is correct.

```
    case "ckb":
      switch (locale.countryCode) {
        case "KU":
          return "کوردی";
        default:
          return locale.languageCode;
      }
```

Comment (SoranTabesh) [reply frontend/appflowy_flutter/packages/flowy_infra/lib/language.dart:25]: case "ckb":
      switch (locale.countryCode) {
        case "KU":
          return "کوردی سۆرانی";
      }

**Comment by LucasXu0** [comment]
@SoranTabesh This pull request conflicts with the main branch. Please submit a new PR to resolve the conflicts.

**Pull request closed without merging**